src/main/resources/jenkins/plugins/http_request/HttpRequest/help-passBuildParameters.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Should build parameters be passed to the URL being called?
+    <p>Should build parameters be passed to the URL being called?</p>
```

Comment:
BTW no `<p>` is needed around single-line help. There is not really a fixed convention though.